runtime/oti/jvmtiInternal.h

```diff
@@ -570,7 +570,17 @@ typedef struct jvmtiGcp_translation {
 #if JAVA_SPEC_VERSION >= 19
 /* These macros corresponds to the states in j.l.VirtualThread. */
 #define JVMTI_VTHREAD_STATE_NEW 0
+#define JVMTI_VTHREAD_STATE_STARTED 1
+#define JVMTI_VTHREAD_STATE_RUNNABLE 2
+#define JVMTI_VTHREAD_STATE_RUNNING 3
+#define JVMTI_VTHREAD_STATE_PARKING 4
+#define JVMTI_VTHREAD_STATE_PARKED 5
+#define JVMTI_VTHREAD_STATE_PINNED 6
+#define JVMTI_VTHREAD_STATE_YIELDING 7
 #define JVMTI_VTHREAD_STATE_TERMINATED 99
+#define JVMTI_VTHREAD_STATE_SUSPENDED (1<<8)
+#define JVMTI_VTHREAD_STATE_RUNNABLE_SUSPENDED (JVMTI_VTHREAD_STATE_RUNNABLE|JVMTI_VTHREAD_STATE_SUSPENDED)
+#define JVMTI_VTHREAD_STATE_PARKED_SUSPENDED (JVMTI_VTHREAD_STATE_PARKED|JVMTI_VTHREAD_STATE_SUSPENDED)
```

Comment:
formatting nitpick: missing space.
```suggestion
#define JVMTI_VTHREAD_STATE_SUSPENDED (1 << 8)
#define JVMTI_VTHREAD_STATE_RUNNABLE_SUSPENDED (JVMTI_VTHREAD_STATE_RUNNABLE | JVMTI_VTHREAD_STATE_SUSPENDED)
#define JVMTI_VTHREAD_STATE_PARKED_SUSPENDED (JVMTI_VTHREAD_STATE_PARKED | JVMTI_VTHREAD_STATE_SUSPENDED)
```

Reply:
Updated.